sei/chain.json

```diff
@@ -519,12 +519,45 @@
           "repo": "https://github.com/sei-ibc-go/v3",
           "version": "v3.3.2"
         }
+      },
+      {
+        "name": "v5.9.0",
+        "proposal": 85,
+        "height": 107000672,
+        "recommended_version": "v5.9.0",
+        "compatible_versions": [
+          "v5.9.0"
+        ],
+        "cosmos_sdk_version": "sei-cosmos v0.3.37-0.20240923023912-aa7a702d42cc",
+        "consensus": {
+          "type": "sei-tendermint",
+          "version": "v0.3.8"
+        },
+        "cosmwasm_version": "sei-wasmd v0.2.4",
+        "cosmwasm_enabled": true,
+        "cosmwasm_path": "$HOME/.sei/wasm",
+        "next_version_name": "",
+        "sdk": {
+          "type": "cosmos",
+          "version": "v0.3.37-0.20240923023912-aa7a702d42cc"
+        },
+        "cosmwasm": {
+          "version": "v0.2.4",
+          "repo": "https://github.com/sei-wasmd",
```

Comment:
I don't think this repo path is valid. What is the org name? I that have to be part of the repo URL